benchmarks/flowertune-llm/evaluation/general-nlp/gen_judgement.py

```diff
@@ -0,0 +1,173 @@
+# This python file is adapted from https://github.com/lm-sys/FastChat/blob/main/fastchat/llm_judge/gen_judgment.py
+
+import argparse
+import json
+from concurrent.futures import ThreadPoolExecutor
+
+import numpy as np
+from fastchat.llm_judge.common import (
+    NEED_REF_CATS,
+    Judge,
+    MatchPair,
+    MatchSingle,
+    check_data,
+    get_model_list,
+    load_judge_prompts,
+    load_model_answers,
+    load_questions,
+    play_a_match_pair,
+    play_a_match_single,
+)
+from fastchat.llm_judge.gen_judgment import (
+    make_judge_pairwise,
+    make_judge_single,
+    make_match,
+    make_match_all_pairs,
+    make_match_single,
+)
+from tqdm import tqdm
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--judge-file",
+        type=str,
+        default="data/judge_prompts.jsonl",
+        help="The file of judge prompts.",
+    )
+    parser.add_argument("--judge_model", type=str, default="gpt-4")
+    parser.add_argument("--baseline_model", type=str, default="gpt-3.5-turbo")
+    parser.add_argument(
+        "--mode",
```

Comment:
I'm thinking it would be better to have the minimum amount of code to run the evaluation. I see this mode is not mentioned in the readme. we always run it as `single` mode? -- or do you think it's better to leave all the models for when we add more evaluation dataset?

Reply:
Yes I think we can simplify this part by removing `mode` argument

Reply:
ok, let's do it